ElasticaBundle/Command/OrchestraCreateSchemaCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace OpenOrchestra\ElasticaBundle\Command;
+
+use LogicException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class OrchestraCreateSchemaCommand
+ */
+class OrchestraCreateSchemaCommand extends ContainerAwareCommand
```

Comment:
the different commands should be more generic with a parameter for specify index for instance